apps/prairielearn/src/lib/editors.ts

```diff
@@ -581,12 +586,18 @@ export class AssessmentCopyEditor extends Editor {
 
   public readonly uuid: string;
 
-  constructor(params: BaseEditorOptions) {
-    super(params);
+  constructor(
+    params: BaseEditorOptions<{ course_instance: CourseInstance; assessment: Assessment }>,
+  ) {
+    const { course_instance, assessment } = params.locals;
+
+    super({
+      ...params,
+      description: `${course_instance.short_name}: copy assessment ${assessment.tid}`,
```

Comment:
Moving `description` to the `Editor` constructor ensures we never forget to set it.